### PR TITLE
fix: use non-reserved exit codes for bcc/bpf install

### PR DESCRIPTION
**Pull request by alexeldeib**
**Reason for Change**:
Most of the exit codes used by the bpf download scripts are reserved. We should avoid using any reserved codes in our scripts. This PR changes all reserved codes to unreserved values.

**Issue Fixed**:
n/a

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by alexeldeib** [comment]
fyi @CecileRobertMichon @mboersma

File: parts/k8s/cloud-init/artifacts/cse_helpers.sh [hunk 1]
```diff
@@ -62,11 +62,11 @@ ERR_AZURE_STACK_GET_NETWORK_CONFIGURATION=121 {{/* Error fetching the network co
 ERR_AZURE_STACK_GET_SUBNET_PREFIX=122 {{/* Error fetching the subnet address prefix for a subnet ID */}}
 
 {{/* BCC/BPF-related error codes */}}
-ERR_IOVISOR_KEY_DOWNLOAD_TIMEOUT=126 {{/* Timeout waiting to download IOVisor repo key */}}
-ERR_IOVISOR_APT_KEY_TIMEOUT=127 {{/* Timeout waiting for IOVisor apt-key */}}
-ERR_BCC_INSTALL_TIMEOUT=128 {{/* Timeout waiting for bcc install */}}
-ERR_BPFTRACE_BIN_DOWNLOAD_FAIL=129 {{/* Failed to download bpftrace binary */}}
-ERR_BPFTRACE_TOOLS_DOWNLOAD_FAIL=130 {{/* Failed to download bpftrace default programs */}}
+ERR_IOVISOR_KEY_DOWNLOAD_TIMEOUT=132 {{/* Timeout waiting to download IOVisor repo key */}}
```
Comment (CecileRobertMichon) [inline parts/k8s/cloud-init/artifacts/cse_helpers.sh]: technically 1 - 2, 126 - 165, and 255 are all reserved so we should avoid that whole range

Comment (alexeldeib) [reply parts/k8s/cloud-init/artifacts/cse_helpers.sh]: missed that all the way up to 165 was reserved for signals. I didn't realize there were so many! fixed, thanks.

**Comment by jackfrancis** [comment]
make generate ftw

**Comment by alexeldeib** [comment]
making a mess left and right...

**Pull request merged**